hbase-assembly/src/main/assembly/hadoop-three-compat.xml

```diff
@@ -143,6 +51,36 @@
   </files>
 
   <dependencySets>
+    <dependencySet>
```

Comment:
Is client.xml and hadoop-three-compat a copy now?

Reply:
Yes.
The differences were already minimal, I think they were accidental, too.

Reply:
Sure, we can see if we should keep both as follow up. I am good as long as generated assembly is same